lib/filters/number.js

```diff
@@ -19,6 +19,53 @@ export function isNumber (value) {
   return _.isNumber(value)
 }
 
+/**
+ * Convert a number into an ordinal numeral.
+ *
+ * @see {@link https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#ordinal-numbers}
+ *
+ * @example
+ * ordinal(4) // fourth
+ * ordinal(22) // 22nd
+ *
+ * @param {number} number - Value to convert
+ * @return {string} Ordinal numeral or `number` with an ordinal suffix
+ */
+export function ordinal (number) {
+  number = _normalize(number, '')
+
+  // Spell out first to ninth
+  if (number < 10) {
+    const ordinals = new Map([
+      [1, 'first'],
+      [2, 'second'],
+      [3, 'third'],
+      [4, 'fourth'],
+      [5, 'fifth'],
+      [6, 'sixth'],
+      [7, 'seventh'],
+      [8, 'eighth'],
+      [9, 'ninth'],
+    ])
+
+    return ordinals.get(number)
+  }
+
+  // After that use 10th, 11th and so on
+  const rule = new Intl.PluralRules('en-GB', {
```

Comment:
Where is `Intl` defined?

Reply:
It’s a global object: [`Intl`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl)